src/com/session/SessionHelper.java

```diff
@@ -15,6 +22,23 @@ public static String currentUser(HttpServletRequest req){
 		}
 	}
 	
+	public static String  isPresent(String email){
```

Comment:
why string as return type, if isPresent, it can possibly have two state, yes or no, you can use boolean to make it simple